mw-webapp/src/component/notification/Notification.tsx

```diff
@@ -0,0 +1,69 @@
+import {ReactElement} from "react";
+import {Cross2Icon} from "@radix-ui/react-icons";
+import {Close, Description, Provider, Root, Viewport} from "@radix-ui/react-toast";
+import styles from "src/component/notification/Notification.module.scss";
+
+/**
+ * Control the sensitivity of the toast for accessibility purposes.
+ * Foreground - for toasts that are the result of a user action.
+ * Background - for toasts generated from background tasks .
+ */
+export enum NotificationType {
+  "foreground" = "foreground",
+  "background" = "background",
+}
+
+/**
+ * Notification props
+ */
+interface NotificationProps {
+
+  /**
+   * The time in milliseconds that should elapse before automatically closing each toast.
+   * @default 5000
+   */
+  duration?: number;
+
+  /**
+   * An author-localized label for each toast.
+   * Used to help screen reader users associate the interruption with a toast.
+   * @default "Notification"
+   */
+  label?: string;
```

Comment:
The description states that the label is "Used to help screen reader users associate the interruption with a toast." this is his only purpose)
I decided that it might be important and decided to add. 
But if you don't think so, I can remove it)